packages/cubejs-query-orchestrator/src/orchestrator/QueryOrchestrator.ts

```diff
@@ -10,6 +10,12 @@ import { LocalQueueEventsBus } from './LocalQueueEventsBus';
 
 export type CacheAndQueryDriverType = 'redis' | 'memory';
 
+export enum DriverType {
+  external = 'external',
+  internal = 'internal',
+  cache = 'cache',
+}
```

Comment:
Usually, enum's keys are capital letters or started from capital letters. Maybe you can update keys?